### PR TITLE
Fetch stocks

### PR DESCRIPTION
**Pull request by tarikbouari**
In this project, I implement the following tasks 

- Filter stocks based on the category on the landing page
- Create detail component and displays information about every crypto
- Create the tests for the application.
- Style your components to match the design provided.
- Refactor code and fix linters error
-  Deploy the project and test for final details.
- Create a good README and PR description.

Here is the link to the video for your project.    [Video Link](https://www.loom.com/share/fb9a755d9f2947a596a3edda764e3b01)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,88 @@
...
+
+**Run from your terminal:**
+
+- `npm test`
+
+_This displays the results of the test suites already written on the project to test for various functionalities and UI implementations. Feel free to edit the UI components and re-run the tests to verify their authenticity_
+
+### Deployment
+
+> coming soon :smile:
```
Comment (Deepakdanger) [inline README.md]: - [ ] **Requirement** The student deploys the application using Heroku, Netlify, or GitHub Pages.
- The project is deployed properly, and the home and detail page works as expected.
- A link to the deployed project is present in the README.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @tarikbouari, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
- [ ] The background color of the tile in the original is the same on the alternate side. kindly change it according to it. In your case, all tile background color is the same.
- Original design 

Screenshot from 2022-10-15 15-58-49

- Your design 
Screenshot from 2022-10-15 15-50-40

- [ ]  One of the component test cases is failing. kindly fix it.
- 
Screenshot from 2022-10-15 15-53-37


- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tarikbouari** [comment]
@Deepakdanger Thanks for your review.  All the changes were made.

File: README.md (hunk 1 above)
Comment (rloterh) [reply README.md]: well resolved! ✔️

File: src/test/component.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import '@testing-library/jest-dom/extend-expect';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Home from '../components/StockList';
+import Nav from '../components/Nav';
+import store from '../redux/configureStore';
+
```
Comment (rloterh) [inline src/test/component.test.js:8]: - [ ] A test case is failing. Kindly ensure to improve upon the same.
image

Comment (rloterh) [inline src/test/component.test.js:8]: - [ ] OPTIONAL
As this activity requires, kindly ensure to include unit tests for this project

File: README.md [hunk 3]
```diff
@@ -1 +1,88 @@
...
+
+## Authors
```
Comment (rloterh) [inline README.md]: - [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Tarike, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- API fetch properly implemented
- Navigation working as intended
- State management with Redux properly implemented

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KDlamini — APPROVED** [review]
### Hi @tarikbouari 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Good API Integration
✔️ Good PR description
✔️ UI follows the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))
✔️ Pages, events, and state management well-implemented
✔️ No linter errors

# Status: Approved ✔️ 

oscars-standing-ovation
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**